Source/Classes/MUMessagesViewController.m

```diff
@@ -54,7 +53,7 @@ - (CGRect) editingRectForBounds:(CGRect)bounds {
     if (rect.origin.x < minx) {
         NSInteger delta = minx - rect.origin.x;
         rect.origin.x += delta;
-        if (MUGetOperatingSystemVersion() < MUMBLE_OS_IOS_7) {
+        if (@available(iOS 7, *)) {} else {
```

Comment:
For the record, XCode shows a warning here if negation is used.

The warning says availability isn't checked properly here - I think the assumption is that you'd use this exclusively for newly introduced APIs. While Swift has an analogous `#unavailable` macro, that wasn't added for Objective-C so there's no way to make the intention clear to avoid that warning.

Therefore, the empty if/else clause here :/